Resources/Prototypes/_Mono/Catalogs/pirate_uplink_catalog.yml

```
@@ -1276,6 +1276,21 @@
       tags:
       - PirateUplink
 
+- type: listing
+  id: UplinkRomerol
+  name: uplink-pirate-romerol-name
+  description: uplink-pirate-romerol-desc
+  productEntity: SyringeRomerol
+  cost:
+    Doubloon: 250
```

i can only imagine this going horribly wrong.
```suggestion
    Doubloon: 350
```